Header header

# 剩余电量百分比
float64 percent
# 输入电压
float64 inputVoltage
# 输入电流
float64 inputCurrent
# 电容电压
float64 capVoltage
# 目标功率
float64 targetPower